diesel/src/result.rs

```diff
@@ -156,6 +156,13 @@ pub trait DatabaseErrorInformation {
     /// Currently this method will return `None` for all backends other than
     /// PostgreSQL.
     fn constraint_name(&self) -> Option<&str>;
+
+    /// An optional integer indicating an error cursor position as an index into
+    /// the original statement string.
+    ///
+    /// Currently this method will return `None` for all backends other than
+    /// PostgreSQL.
```

Comment:
I wouldn't add such statements to the documentation, because we cannot possibly know how third party backends handle this.